src/main/java/duke/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class is a subclass of Task.
+ * @author Sim Jun Heng
+ * @version CS2103T AY21/22 Sem 2
+ */
+public class Event extends Task {
+
+    private LocalDate date;
+
+    /**
+     * Assigns desc, date and done to this instance.
+     *
+     * @param desc the task description
+     * @param date the date of the event
+     * @param isDone the current completion status of the task.
+     */
+    public Event(String desc, LocalDate date, boolean isDone) {
+        super(desc, isDone);
+        this.date = date;
+    }
+
+    @Override
+    public String toString() {
+        String str = "[E]";
+        // Convert Current Date Format to MMM dd yyyy
+        String customPattern = "MMM dd yyyy";
+        DateTimeFormatter customFormat = DateTimeFormatter.ofPattern(customPattern);
```

Comment:
Just a suggestion, but in my opinion, I think you can skip line 31 and directly alter line 32 to be `DateTimeFormatter customFormat = DateTimeFormatter.ofPattern("MMM dd yyyy");`

I don't think it would break the logical flow or make it less comprehensive. Maybe this is something you can consider if you agree :)